<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Track View</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <zorder>groupBox</zorder>
          <zorder>groupBox</zorder>
          <zorder>groupBox</zorder>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Filter Tracks</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Mean Distance Per Lag Maximum: </string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>20</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Neighborly Distance:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Mean Distance Per Lag Minimum</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>50</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Required Neighbors:</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="MLDMinimumSpin">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>20</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="MLDMaximumSpin">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>50</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QSpinBox" name="neighborSpin">
           <property name="geometry">
            <rect>
             <x>471</x>
             <y>50</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="neighborDistanceSpin">
           <property name="geometry">
            <rect>
             <x>470</x>
             <y>20</y>
             <width>62</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QCheckBox" name="hideBackgroundCheck">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>20</y>
             <width>151</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Hide Background Image</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="ignoreOutsideCheck">
           <property name="geometry">
            <rect>
             <x>570</x>
             <y>50</y>
             <width>151</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Ignore Tracks Outside ROIs</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Track Analysis</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_2" native="true"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImportBin"/>
    <addaction name="actionImportBackground"/>
    <addaction name="actionImportCoordinates"/>
   </widget>
   <widget class="QMenu" name="menuSimulate">
    <property name="title">
     <string>Simulate</string>
    </property>
    <addaction name="actionSimulateDistances"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExportMSD"/>
    <addaction name="actionExportHistogram"/>
    <addaction name="actionExportOutlined"/>
    <addaction name="actionExportDistances"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulate"/>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportBin">
   <property name="text">
    <string>Import .bin file</string>
   </property>
  </action>
  <action name="actionImportBackground">
   <property name="text">
    <string>Import Background Image</string>
   </property>
  </action>
  <action name="actionImportCoordinates">
   <property name="text">
    <string>Import Coordinates</string>
   </property>
  </action>
  <action name="actionSimulateDistances">
   <property name="text">
    <string>Simulate Mean Distances</string>
   </property>
  </action>
  <action name="actionExportMSD">
   <property name="text">
    <string>Export MSD Values</string>
   </property>
  </action>
  <action name="actionExportHistogram">
   <property name="text">
    <string>Export Histogram Values</string>
   </property>
  </action>
  <action name="actionExportOutlined">
   <property name="text">
    <string>Export Plotted Tracks</string>
   </property>
  </action>
  <action name="actionExportDistances">
   <property name="text">
    <string>Export All Distances</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
